spec/rdkafka/admin_spec.rb

```diff
@@ -19,6 +19,15 @@
   let(:topic_config)             { {"cleanup.policy" => "compact", "min.cleanable.dirty.ratio" => 0.8} }
   let(:invalid_topic_config)     { {"cleeeeenup.policee" => "campact"} }
 
+
+  let(:resource_type)         {2}
```

Comment:
Could we use the constants from `bindings.rb` here instead of magic values?

Reply:
fixed